api/src/data/migrations/047.create-data-injection-sources-table.ts

```diff
@@ -0,0 +1,13 @@
+import * as knex from "knex";
```

Comment:
These migrations name should be using timestamps rather than numbered values.
Why? Because that way you are unlikely to run into conflicts with other developers working on the same project.